distribution/pom.xml

```diff
@@ -108,6 +108,8 @@
                                 <argument>io.druid.extensions:druid-stats</argument>
                                 <argument>-c</argument>
                                 <argument>io.druid.extensions:druid-examples</argument>
+                                <argument>-c</argument>
+                                <argument>io.druid.extensions:sample-client-sslcontext</argument>
```

Comment:
is the sample-client-sslcontext intended to be used in production or users are encouraged to implement their own extensions ? 
If its intended for production use maybe drop the keyword 'sample' ?